examples/internal/proto/examplepb/echo_service.proto

```diff
@@ -69,4 +83,11 @@ service EchoService {
 			delete: "/v1/example/echo_delete"
 		};
 	}
+	// EchoPatch method receives a NonStandardUpdateRequest and returns it.
+	rpc EchoPatch(DynamicMessageUpdate) returns (DynamicMessageUpdate) {
+		option (google.api.http) = {
+			patch: "/v1/example/echo_patch"
+      body: "body"
```

Comment:
I think we ended up mixing tabs and spaces here.

Reply:
Fixed. Thank you.